From 2899bbe37ac8399503a735db68a41071811ded85 Mon Sep 17 00:00:00 2001
From: Gurchetan Singh <gurchetansingh@chromium.org>
Date: Wed, 25 Sep 2019 09:25:46 -0700
Subject: [PATCH 1/1] virgl: remove stride from virgl_hw_res

It's not used anywhere, and stride isn't really an intrinsic
property of a GEM buffer.

Reviewed by: Robert Tarasov <tutankhamen@chromium.org>
---
 src/gallium/winsys/virgl/drm/virgl_drm_winsys.c | 2 --
 src/gallium/winsys/virgl/drm/virgl_drm_winsys.h | 1 -
 2 files changed, 3 deletions(-)

diff --git a/src/gallium/winsys/virgl/drm/virgl_drm_winsys.c b/src/gallium/winsys/virgl/drm/virgl_drm_winsys.c
index cc4371fb5fdd..741064a958cd 100644
--- a/src/gallium/winsys/virgl/drm/virgl_drm_winsys.c
+++ b/src/gallium/winsys/virgl/drm/virgl_drm_winsys.c
@@ -190,7 +190,6 @@ virgl_drm_winsys_resource_create(struct virgl_winsys *qws,
    res->res_handle = createcmd.res_handle;
    res->bo_handle = createcmd.bo_handle;
    res->size = size;
-   res->stride = stride;
    pipe_reference_init(&res->reference, 1);
    p_atomic_set(&res->external, false);
    p_atomic_set(&res->num_cs_references, 0);
@@ -374,7 +373,6 @@ virgl_drm_winsys_resource_create_handle(struct virgl_winsys *qws,
    res->res_handle = info_arg.res_handle;
 
    res->size = info_arg.size;
-   res->stride = info_arg.stride;
    pipe_reference_init(&res->reference, 1);
    p_atomic_set(&res->external, true);
    res->num_cs_references = 0;
diff --git a/src/gallium/winsys/virgl/drm/virgl_drm_winsys.h b/src/gallium/winsys/virgl/drm/virgl_drm_winsys.h
index 9c9d0a5a708a..3a2c476a00b0 100644
--- a/src/gallium/winsys/virgl/drm/virgl_drm_winsys.h
+++ b/src/gallium/winsys/virgl/drm/virgl_drm_winsys.h
@@ -41,7 +41,6 @@ struct virgl_hw_res {
    int num_cs_references;
    uint32_t size;
    void *ptr;
-   uint32_t stride;
 
    struct virgl_resource_cache_entry cache_entry;
    uint32_t bind;
-- 
2.24.0.rc0.303.g954a862665-goog

